core/src/test/scala/integration/kafka/api/BaseQuotaTest.scala

```diff
@@ -57,9 +57,9 @@ abstract class BaseQuotaTest extends IntegrationTestHarness {
   this.consumerConfig.setProperty(ConsumerConfig.FETCH_MAX_WAIT_MS_CONFIG, "0")
 
   // Low enough quota that a producer sending a small payload in a tight loop should get throttled
-  val defaultProducerQuota = 8000
-  val defaultConsumerQuota = 2500
-  val defaultRequestQuota = Int.MaxValue
+  val defaultProducerQuota: Long = 8000
+  val defaultConsumerQuota: Long = 2500
+  val defaultRequestQuota: Double = Long.MaxValue
```

Comment:
Can you please add a `toDouble` call here? This silent conversion triggers a warning in Scala 2.13.

Reply:
Same for other cases where we are passing a `Long` when a `Double` is expected.

Reply:
Sure, done.